services/galley/src/Galley/API.hs

```diff
@@ -293,17 +296,27 @@ sitemap = do
       .&. capture "tid"
       .&. capture "uid"
       .&. accept "application" "json"
+  -- This endpoint can lead to the following events being sent to clients:
```

Comment:
to whose clients?

Reply:
To all clients belonging to the people listed in the lines below, in this case the "contacts of the user" mentioned in the request (i.e. `:uid`). Would it be clearer to write "LegalHoldClientRequested event to contacts of the user the client is requested for"?

I use a similar wording everywhere, talking about events being sent to clients, to make clear that clients will receive them directly, as opposed to some server (think for example other backends with federation). Do you have any suggestions for better wording? Maybe "sent to all clients of the specified users:"?

Reply:
I'll just out the "to clients" part.

Reply:
done